docs/learn/tokenomics2/Inflation.md

```diff
@@ -2,33 +2,166 @@
+
+_Collators_ get a fixed amount of the cycle's _soft-capped_ inflation.
+This amount is equally divided by the number of blocks in the cycle.
+
+$collator\_reward\_per\_block = \frac{total\_collator\_cycle\_reward}{blocks\_per\_cycle}$
+
+### Treasury
+
+Similar to the _collators_, treasury gets a fixed amount of the cycle's _soft-capped_ inflation.
+To avoid it being minted all at once, the amount is equaly divided by the number of blocks in the cycle.
+
+$treasury\_reward\_per\_block = \frac{total\_treasury\_cycle\_reward}{blocks\_per\_cycle}$
+
+### dApps
+
+dApp reward are _assigned_ at the end of each era during `Build&Earn` subperiod.
+This means that the total cycle's dApp reward amount has to be equally divided by the total number of `Build&Earn` eras in a cycle.
+
+$dapp\_reward\_pool\_per\_era = \frac{total\_dapp\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The dApp staking protocol will calculate how much each staked dApp should get.
+
+### Stakers
+
+There are two components to the staker rewards - regular _staking_ rewards & the _bonus_ reward for loyal stakers.
+
+#### Regular Staker Rewards
+
+Regular staker rewards are awarded for staking native currency, **ASTR**, on a dApp.
+These rewards have two components - the _base_ reward and the _adjustable_ reward.
+
+Base reward is the amount assigned to the reward pool regardless of how much has been staked in total.
+
+$base\_staker\_reward\_pool\_per\_era = \frac{total\_base\_staker\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The adjustable part is the dynamic part, and depends on the _total value staked_ and the _target stake value_.
+This amount linearly increases as the _total value staked_ increases, and then saturates once the amount is reached or exceeded.
+With this component, _staker rewards_ are not a _zero-sum game_.
+
+$max_\_adjustable\_staker\_reward\_pool\_per\_era = \frac{total\_adjustable\_staker\_cycle\_reward}{number\_of\_cycles * eras\_per\_build\_and\_earn}$
+
+The adjustable part of the reward is calculated once an era ends, using the _total value staked_ at that point in time.
+
+$adjustable\_factor = min(1, \frac{total\_value\_staked\_percent}{ideal\_staking\_percent})$
+
+Using the _adjustable\_factor_, adjustable portion of the staker reward is:
+
+$adjustable\_staker\_reward\_pool = max_\_adjustable\_staker\_reward\_pool\_per\_era * adjustable\_factor$
+
+When the _adjustable factor_ is less than **1**, it means the remainder is never minted, reducing the overall inflation.
+
+With the above formulas, we can finally express how much staker _Alice_ earns in era **n**:
+
+$staker\_reward_{Alice} = \frac{staked\_value_{Alice,n}}{total\_staked\_value_n} * (base\_staker\_reward\_pool\_per\_era_n + adjustable\_staker\_reward\_pool_n)$
+
+#### Bonus Rewards
+
+In case a staker stakes during the `Voting` subperiod, and doesn't reduce their stake below what was staked at the end of the `Voting` subperiod,
```

Comment:
suggestion: add (during `Build&Earn`) to sentence:

In case a staker stakes during the `Voting` subperiod, and doesn't reduce their stake (during `Build&Earn`) below what was staked at the end of the `Voting` subperiod,

Reply:
Applied.